content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,180 @@
+=======================================================
+Connecting the Windows IoT software to an Odoo database
+=======================================================
+
+A Virtual IoT box is a computer program that needs to be downloaded and installed on a Windows
```

Comment:
Unless it is a proper noun or is the name of the tool/module we shouldn't capitalize *virtual* in virtual IoT box. Looks like there are only a few of these throughout the doc that have the `V` capitalized :)
```suggestion
A virtual IoT box is a computer program that needs to be downloaded and installed on a Windows
```